src/components/WwwFrame/TheHeader.vue

```diff
@@ -345,66 +345,41 @@
 								}"
 							>
 								<router-link
-									v-if="enableBasketExperiment"
```

Comment:
Instead of removing the fallback, we should be using the feature flag here. We want a way to roll this back on prod immediately if there are issues, like we have in place for the TY page. Rolling out new features to new users without fallback is dangerous when it's such a critical part of the app. Here's the recent ticket that added the feature flag: https://kiva.atlassian.net/browse/MP-1322

Reply:
Oh, right, I misread the code, this code isn't specific to the experiment.